### ISSUE TITLE
palera1n_2.0-iphoneos-arm64

### ISSUE THREAD

**Issue by Byronetyrone** [palera1n/palera1n#451]
### Feature Description

can you compile an  iphone arm 64 binary or deb like you did with all the previous releases? the beta 7 version works pretty good to jailbreak my iphonex in a pinch using my phone with dopamine rootless jailbreak. would be nice to test the 2.0 version since some things were fixed regarding ios16.x thanks

### Start going into detail..

deb would be really cool but just the binary is fine.

**Comment by khcrysalis** [comment]
We decided to remove it originally due it not working most of the time, though you can find the builds here for 2.0 https://cdn.nickchan.lol/palera1n/artifacts/c-rewrite/v2.0/470/

In future we may consider re-adding it.

**Comment by Byronetyrone** [comment]
nice thanks. really the only part that doesn’t work that well is getting the “trust this computer” dialog to pop up.  i’m sure there’s a command i could type to restart something to make it work.. usbmuxd maybe idk. doesn’t seem like a lot of people try to do this

**Comment by BerlinSnoop** [comment]
Yes, please re-add it. I am doing the same… using my dopamined iphone 14pro to rejailbreak my ipad 2pro.

**Comment by misterjrw** [comment]
@Byronetyrone, @BerlinSnoop interested in your comments as I'm trying to do the same with an old iphone 6s+.

Everything is going great, paired device, usbmuxd installed so 6s+ puts iphone 8 (16.7.10) into recovery, so I know the Apple Camera Adapter is working. 

Then I successfully put the 8 into DFU and at that point, I get an Accessory Firmware Upgrade warning which if I press OK, palera1n fails with either a) failed to issue transaction abort, or b) failed to issue asynchronous download. It's as if the data can't be sent from the 6s+ for one reason or another.

I'm assuming it's down to the Apple Adapter, so purchased another (non-Apple but MFI certified) and it does the same.

Could I ask what you use to connect your two idevices together please?

I have an old Android and I can successfully re-jailbreak the 8 using terminal in TWRP but I would like to use the 6s+ if at all possible.

Thanks for any help you can provide.

**Comment by BerlinSnoop** [comment]
I am using an usb hub with external power supply. This works. Sometimes palerai!n hangs but if you kill it an restart it then it runs through.

**Comment by misterjrw** [comment]
Hi and thank you so much for responding.

I will try what you suggested, to be honest, it had crossed my mind but I wanted this method to be as portable as possible. But if it needs a powered hub, then it needs a powered hub!!

Cheers

Jonathan
